security/pkg/k8s/controller/workloadsecret.go

```diff
@@ -41,6 +41,10 @@ const (
 	// The Istio secret annotation type
 	IstioSecretType = "istio.io/key-and-cert"
 
+	// Label which determines whether Citadel should watch namespace
+	NamespaceManagedLabel = "ca.istio.io/secret-controller-namespace"
+	NamespaceIgnoreValue  = "NONE"
```

Comment:
What is this and why ? Comments, etc.

Reply:
If the `NamespaceManagedLabel` has the value of `NamespaceIgnoreValue`, all Citadels will ignore the namespace. Note that the Citadels will ignore the namespace if the label has _any_ value that doesn't match its namespace, but it makes sense to have a designated one as well.

This extra magic-value for disabling a NS across all Citadels is admittedly a bit clunky, would be open to suggestions.